README.md

```diff
@@ -56,14 +56,56 @@ If you want to follow the updates or learn more about the latest design then joi
 The Cosmos ecosystem is vast.
 [Awesome Cosmos](https://github.com/cosmos/awesome-cosmos) is a community-curated list of notable frameworks, modules and tools.
 
-### Cosmos Hub Mainnet
-
-The Cosmos Hub application, `gaia`, has its own [cosmos/gaia repository](https://github.com/cosmos/gaia). Go there to join the Cosmos Hub mainnet and more.
-
 ### Inter-Blockchain Communication (IBC)
 
 The IBC module for the Cosmos SDK has its own [cosmos/ibc-go repository](https://github.com/cosmos/ibc-go). Go there to build and integrate with the IBC module.
 
+### Version Matrix
+
+The version matrix below shows which versions of the Cosmos SDK, modules and libraries are compatible with each other.
+
+#### Core Dependencies
+
+Core Dependencies are the core libraries that a application may depend on. 
+
+> Note: the ❌ signals that the version of the Cosmos SDK does not need to import the dependency.
+
+| Cosmos SDK | cosmossdk.io/core | cosmossdk.io/api | cosmossdk.io/math | cosmossdk.io/errors | cosmossdk.io/depinject | cosmossdk.io/log | cosmossdk.io/store |
+|------------|-------------------|------------------|-------------------|---------------------|------------------------|------------------|--------------------|
+| 0.50.x     | 0.11.x            | 0.7.x            | 1.x.x             | 1.x.x               | 1.x.x                  | 1.x.x            | 1.x.x              |
+| 0.47.x     | 0.5.x             | 0.3.x            | 1.x.x             | 1.x.x               | 1.x.x                  | 1.x.x            | ❌                  |
+| 0.46.x     | ❌                 | ❌                | 1.x.x             | 1.x.x               | ❌                      | ❌                | ❌                  |
+
+#### Module Dependencies
+
+Module Dependencies are the modules that a application may depend on and which version of the Cosmos SDK they are compatible with.
+
+> Note: The version table only goes back to 0.50.x, this is due to the reason that modules were not spun out into their own go.mods until 0.50.x. 
+
+
+| Cosmos SDK                  | 0.50.x    | 0.y.z |
```

Comment:
We only have 4 modules extracted for v0.50 (circuit, evidence, feegrant, upgrafe and nft)
x/tx as well if we count this as a module.

Reply:
thank you